config/routes.rb

```diff
@@ -1,4 +1,21 @@
 Rails.application.routes.draw do
+  # TPI
+  resources :sectors, only: [:show, :index] do
```

Comment:
I think TPI and the future Laws website should be namespaced, as later they will sit in different domains, and might be easiest to link to /tpi or /laws, and then keep the routes isolated. Wondering if we should do that straight away or later when is actually needed. What do you think?

Reply:
good point, I can add `/tpi` namespace now

Reply:
d6efe5a